src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;netcoreapp3.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;netstandard2.0;netcoreapp3.0;net461</TargetFrameworks>
```

Comment:
I think it will only output an `AnyCPU` assembly if either the `TargetFrameworks` doesn't have browser or the Architectures aren't included to have `wasm` which will mean `'$(TargetArchitecture)' == 'wasm'` will never be true?